generate-summary.py

```diff
@@ -26,26 +29,35 @@
 		try:
 			idx = int(v[0][1:])
 		except:
-			print 111, v
-			print 112, v[0]
+			print(111, v)
+			print(112, v[0])
 		if idx != i:
-			print "missing value!", idx, i
+			print("missing value!", idx, i)
 		val = int(v[1])
 		try:
 			total[idx-1][1][val] += 1
 		except:
-			print len(total), idx
+			print(len(total), idx)
 		i += 1
 
 # just copy values to jure.xls
+result = []
 for t in total:
 	cnt = sum(t[1])
 	if cnt == 0:
 		continue
 	if rel:
 		out=""
+		out2 = []
 		for c in t[1]:
 			out+= str(float(c*100) / float(cnt))[:4] + " " 
-		print out
+			out2.append(round(((c*100)/cnt),1))
+		print(out)
+		result.append(out2)
 	else:
-		print t[0], t[1], cnt
+		print(t[0], t[1], cnt)
```

Comment:
This branch is missing addition to the file writing list.

Reply:
would I be correct to say that if <code>rel=False</code>, then we don't want the percentages but the absolute numbers? In this case, do we want the cat-shares.csv file to be overwritten with absolute numbers or a separate file to be created?

Reply:
Yes and the same file is fine, since it is trivial to regenerate. This absolute mode is only used rarely, when there's a need for some base statistics.